java-frontend/src/main/java/org/sonar/java/resolve/JavaType.java

```diff
@@ -62,6 +62,9 @@
   JavaSymbol.TypeJavaSymbol symbol;
 
   public JavaType(int tag, JavaSymbol.TypeJavaSymbol symbol) {
+    Random r = new Random();
+    double rand = r.nextDouble() * 50;  // Noncompliant way to get a pseudo-random value between 0 and 50
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "rand" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1481)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)